api/scripts/wazuh-apid.py

```diff
@@ -113,26 +132,28 @@ def start(foreground, root, config_file):
                   swagger_from_file=os.path.join(app.specification_dir, 'spec.yaml'))
 
     # Configure https
-    if api_config['https']['enabled']:
+    if configuration.api_conf['https']['enabled']:
         try:
             ssl_context = ssl.SSLContext()
-            if api_config['https']['use_ca']:
+            if configuration.api_conf['https']['use_ca']:
                 ssl_context.verify_mode = ssl.CERT_REQUIRED
-                ssl_context.load_verify_locations(api_config['https']['ca'])
-            ssl_context.load_cert_chain(certfile=api_config['https']['cert'],
-                                        keyfile=api_config['https']['key'])
+                ssl_context.load_verify_locations(configuration.api_conf['https']['ca'])
+            ssl_context.load_cert_chain(certfile=configuration.api_conf['https']['cert'],
+                                        keyfile=configuration.api_conf['https']['key'])
         except ssl.SSLError as e:
             raise APIException(2003, details='Private key does not match with the certificate')
         except IOError as e:
-            raise APIException(2003, details='Please, ensure '
+            raise APIException(2003, details=f'{e} - Please, ensure '
                                              'if path to certificates is correct in '
                                              'the configuration file '
                                              f'(WAZUH_PATH/{to_relative_path(CONFIG_FILE_PATH)})')
     else:
         ssl_context = None
 
-    app.run(port=api_config['port'],
-            host=api_config['host'],
+    pyDaemonModule.create_pid('wazuh-apid', os.getpid())
```

Comment:
This line must be removed. Currently wazuh-api service does not need any pid file.